gnome-base/dconf/dconf-0.24.0-r1.ebuild

```diff
@@ -14,10 +14,12 @@ LICENSE="LGPL-2.1+"
 SLOT="0"
 
 # TODO: coverage ?
-IUSE="test"
+IUSE="test X"
 
 KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~x86-fbsd ~arm-linux ~x86-linux"
 
+PDEPEND="X? ( gnome-base/dconf-editor )"
```

Comment:
This sound like a nice idea for migration but we avoid pure runtime dependencies controlled by USE flags as explained in the dev manual. We probably want to take note of this transition in a Gnome 3.16 migration guide on the Gentoo wiki.